package-lock.json

```diff
@@ -8,11 +8,16 @@
       "name": "movie-quotes",
       "version": "0.1.0",
       "dependencies": {
+        "@headlessui/react": "^1.6.6",
+        "@heroicons/react": "^1.0.6",
+        "framer-motion": "^6.5.1",
+        "heroicons": "^1.0.6",
         "i18next": "^21.8.14",
         "next": "12.2.3",
         "next-i18next": "^11.3.0",
         "react": "18.2.0",
-        "react-dom": "18.2.0"
+        "react-dom": "18.2.0",
+        "react-parallax": "^3.5.1"
```

Comment:
oh come on now, its not that hard to make with vanila css, do not add uneeded dependancies for no reason, check this: https://codepen.io/HenryZarza/pen/VwYagOO

Reply:
I did with vanilla in the end, I didn't like that package